Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ccl/configuration/update/CCLConfigurationUpdateScheduler.kt

```diff
@@ -48,14 +47,13 @@ class CCLConfigurationUpdateScheduler @Inject constructor(
     }
 
     private fun buildWorkRequest(): PeriodicWorkRequest {
-        val constraints = Constraints.Builder()
-            .setRequiredNetworkType(NetworkType.CONNECTED)
-            .build()
-
         return PeriodicWorkRequestBuilder<CCLConfigurationUpdateWorker>(
             repeatInterval = 24,
             repeatIntervalTimeUnit = TimeUnit.HOURS
-        ).setConstraints(constraints).build()
+        ).setInitialDelay(
```

Comment:
Not related to the bug, but I added a 1 minute delay for the worker so that the app and the worker won't both update the config after app start at the same time.